code/modules/economy/traders/generic.dm

```diff
@@ -181,13 +192,29 @@
 /datum/commodity/trader/generic/shipcomponents
 	comname = "Ship/Pod Components"
 	comtype = /obj/item/shipcomponent/
-	price_boundary = list(500,2500)
+	price_boundary = list(300,1000)
 	possible_names = list("Our shipyards are low on ship-building components. Could you sell us some?",
-	"We're trying a new experimental pod construction technique and need all the ship components we can get.")
+	"We're trying a new experimental pod construction technique and need all the ship components we can get.",
+	"Some jokers sold us 600 cargo bays when we asked for parts, so we can't build any pods. We're in dire need of some other ship components.")
 
 /datum/commodity/trader/generic/jumpsuits
 	comname = "Jumpsuits"
 	comtype = /obj/item/clothing/under/
-	price_boundary = list(40,100)
+	price_boundary = list(100,200)
 	possible_names = list("We're drafting in some new staff soon, and need new jumpsuits.",
 	"We need any jumpsuits you can spare. Don't ask.")
+
+
+/datum/commodity/trader/generic/furniture
+	comname = "Furniture Parts"
+	comtype = /obj/item/furniture_parts/
+	price_boundary = list(100, 300)
+	possible_names = list("We need some more furniture to spice up our ship.",
+	"We're building a new lounge, and we need whatever furniture you have.")
+
+/datum/commodity/trader/generic/pens
+	comname = "Writing Utensils"
+	comtype = /obj/item/pen/
```

Comment:
not pencils as well?

Reply:
https://github.com/goonstation/goonstation/blob/72a8bd2719fa2cda5533f45f31573842d947ac1a/code/obj/item/pens_writing_etc.dm#L162
uhhh

Reply:
lmaooo